lib/sdk/storage/datauri.js

```diff
@@ -27,7 +28,39 @@ async function download(source, file) {
         throw new GenericError(err.message, `${actionName()}_download`);
     }
 }
-
+/**
+ * Generate Presign Url taking local file path as input
+ * Creates a unique name for the file, upload the file to 
+ * storage from the path provided and generates a PreSigned Url
+ * @param {String} Path to local file system, where file is available
+ * @param {Number} Attempt, this indicates retry attempt, used for mock
+ * @returns {String} PreSigned Url of upload the source file
+ */
+async function getPreSignedUrl(source) {
```

Comment:
I think we already havve this function here: https://github.com/adobe/asset-compute-pipeline/blob/master/lib/engine.js#L446-L465
it would be good to somehow only have this in one place

Reply:
good catch, keeping the one from datauri since it is under sdk/ folder

Reply:
Ack, fine for now. I think when we export the shared code we should move to this storage.js. can you add a note about this?